Automattic/wp-calypso
# A4A: partner directory fix language submission

Related to p1718322425504479/1718321725.021229-slack-C047ACYM8UQ

## Proposed Changes

Fix Partner details form submission (languages)

## Why are these changes being made?



## Testing Instructions



- Using live link, navigate to `partner-directory/agency-details` and check that you are able to submit form normally and languages are represented correctly

## Pre-merge Checklist
